src/ocex.rs

```diff
@@ -22,17 +22,21 @@ pub enum IngressMessages<AccountId, Balance> {
     RemoveProxy(AccountId, AccountId),
 }
 
-#[derive(Clone, Encode, Decode, MaxEncodedLen, TypeInfo, Debug, PartialEq)]
+#[derive(Clone, Encode, Decode,     MaxEncodedLen, TypeInfo, Debug, PartialEq)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub enum EgressMessages<AccountId, Balance> {
+pub enum EgressMessages<AccountId, Balance, String> {
```

Comment:
What would be the expected type for this Generic?

Reply:
A string that is basically serialized contains a snapshot data struct.